### DISCUSSION TITLE
Responding to emails from a different mailbox than messages are sent to.

### DISCUSSION THREAD

**Discussion by ITechGeek81** [simple-login/app#475] [Feature Requests]
So I have a Google Workspace account w/ 1 address as my login account and a bunch of aliases.  The stuff forwarded through Simplelogin is routed to one of the aliases.

When I respond to emails sent through Simplelogin (unless I add the login account as one of the mailboxes), I get a bounce back saying unauthorized email address even through the from address on the email is the same as the address it was sent to, it seems Simplelogin isn't looking at the from address in the headers, but the return-path.  Is it possible to get an option to change that?  Or create a whitelist of addresses to send from that can go through all aliases?

**Comment by nguyenkims** [comment]
The response is checked using the *envelope* mail_from which is harder to fake than the From: header. This envelope mail_from is also often set as the `return-path` header. 

If you want to allow the address used for sending out emails, you can add it as "Authorized addresses" on the mailbox page (cf the below screenshot)

![authorized-address](https://user-images.githubusercontent.com/3167780/118669775-53298580-b7f6-11eb-941f-65c78fccd18d.png)

**Reply by ITechGeek81** [reply]
Thank you, I think this is what I needed.
